### ISSUE TITLE
Time to fork tgenv?

### ISSUE THREAD

**Issue by iainelder** [gruntwork-io/terragrunt#1695]
tgenv is a community tool for installing multiple versions of Terragrunt on the same machine and switching between them automatically per project.

[Gruntwork itself recommends the tool](https://blog.gruntwork.io/how-to-manage-multiple-versions-of-terragrunt-and-terraform-as-a-team-in-your-iac-project-da5b59209f2d), although it until now has depended on others for its maintenance.

There are at least two competing implementations of tgenv, and each appears to be abandoned by its authors:

* https://github.com/taosmountain/tgenv
* https://github.com/cunymatthieu/tgenv

PRs submitted by its users to add new features or fix bugs go unanswered and unmerged.

It seems like it's in Gruntwork's interest to ensure that tgenv continues to receive maintenance from its user community.

To ensure the conuinued availabliity and reliablity of the tool, would Gruntwork be willing to fork one of these versions of the tool?

(My personal preference is for toasmountain's version, because of a bug in the output command for cunymatthieu's version https://github.com/cunymatthieu/tgenv/issues/11. But it wouldn't matter if Prs would be accepted to fix the bugs :-D)

**Comment by brikis98** [comment]
Yes, we looked into this question in https://github.com/cunymatthieu/tgenv/issues/19. 

It sounds like `tgenv` isn't maintained, so we've started using [`tgswitch`](https://github.com/warrensbox/tgswitch). It seems to work well enough, so we may just transition to that. Of course, we'd love to take over and maintain some project, but the reality is we don't have a lot of resources to spare right now, so if we can lean on good projects maintained by others, that's probably the best bet for now.
